SleepEngine/SleepEngine.vcxproj

```diff
@@ -263,15 +269,23 @@
     <ClInclude Include="Engine\Utils\file_utils.h" />
     <ClInclude Include="Engine\Utils\json_serialize_utils.h" />
     <ClInclude Include="Engine\Utils\MacroUtils.h" />
+    <ClInclude Include="Engine\Utils\math.h" />
+    <ClInclude Include="Engine\Utils\stl_utils.h" />
     <ClInclude Include="Engine\Utils\unit_utils.h" />
     <ClInclude Include="stdafx.h" />
   </ItemGroup>
   <ItemGroup>
+    <None Include="Demos\Bubbles\Config\bubble_config.json" />
     <None Include="Engine\Config\config.json" />
     <None Include="Engine\Render\Shaders\shader.fs" />
     <None Include="Engine\Render\Shaders\shader.vs" />
   </ItemGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
   <ImportGroup Label="ExtensionTargets">
   </ImportGroup>
+  <ProjectExtensions>
+    <VisualStudio>
+      <UserProperties Demos_4Bubbles_4Config_4bubble_3config_1json__JSONSchema="http://json.schemastore.org/renovate" />
+    </VisualStudio>
+  </ProjectExtensions>

```

Comment:
What is this?

Reply:
dunno, some vs metadata for json files

Reply:
Have you used any visual studio extensions for JSON?

Reply:
nope

Reply:
Oh shit